### DISCUSSION TITLE
Is there a visual or screenshot showing the jobs in que or active in this app?

### DISCUSSION THREAD

**Discussion by datatalking** [agronholm/apscheduler#784] [Q&A]
I think the apscheduler app might solve issues for three projects I have right now, and I was looking for screenshots or images of what you see when it runs.

1. I couldn't find any images after reviewing past discussions so here is a sample image of when conda updates within pycharm I get this txt update page which helps.
![Screen Shot 2023-08-29 at 12 41 11 PM](https://github.com/agronholm/apscheduler/assets/11813209/146647d8-4a2f-47fb-a36c-a7254d4860ca)

2. That got me wondering if apscheduler is headless is this an app that I can get a list of tasks_in_que, tasks_in_processcan and tasks_completed or perhaps tasks_not_completed?
3. I'm willing to submit PR to improve code if these options are available or help create them if not available but they are in line with your current issues or build goals.
4. Could add a basic dashboard or create something even better than has a little decision in intelligence to communicate back to me that tasks were completed like Github workflows does.

**Comment by agronholm** [comment]
It's a library, and as such has no visuals. Third party projects may implement user interfaces for it (this is mentioned in a [FAQ entry](https://apscheduler.readthedocs.io/en/3.x/faq.html#is-there-a-graphical-user-interface-for-apscheduler)).

As for item 2, there is unfortunately no sophisticated method for this. APScheduler 3.x has `get_jobs()`, but even that doesn't show the status (whether it's running, pending or done). I will attempt to improve on this in the APScheduler 4 series.

**Comment by datatalking** [comment]
@agronholm thank you for the reply. What would be the output of or getting a list of jobs that it currently has by date or score?

```
print(get_jobs('txt'))
```

What would that do if you get what I mean?
